integration-tests/utils/integration-utils.js

```diff
@@ -141,35 +142,37 @@ const openError = async page => {
   return !!errorDetails.length
 }
 
+const maybeSaveScreenshot = async (page, testName) => {
+  if (!screenshotDir) { return }
```

Comment:
Not sure how/if ever `screenshotDir` may be `falsy` but I wonder if we should log info in case it is. Perhaps something like:

```suggestion
  if (!screenshotDir) { 
    console.log(`Cannot take screenshot as screenshot directory doesn't exist.`) 
    return
  }
```

Reply:
I think I should rename this variable to `screenshotDirPath`, since it is only a string. The directory is created before the tests are started: https://github.com/DataBiosphere/terra-ui/blob/dev/.circleci/config.yml#L157

The environment variable `SCREENSHOT_DIR` that gets assigned to this variable is serving two purposes: To tell us where to put the screenshots and as a flag to enable screenshot capturing via this conditional.

Would the rename help clarify?

Reply:
Renaming the variable as `screenshotDirPath` makes sense.

Are you also saying, we don't need to log a message? If so, I'm not sure I understand why we don't.